packages/uikit/src/components/Select/Select.tsx

```diff
@@ -30,10 +30,6 @@ const DropDownListContainer = styled.div`
   transform-origin: top;
   opacity: 0;
   width: 100%;
-
-  ${({ theme }) => theme.mediaQueries.sm} {
```

Comment:
Is this going to affect any other place that uses this component?

Reply:
hmm i did check on farms and it didnt break there form my checks, but havent check anywhere else. i will look at all locations where Select is used tomorrow. but i think this will be fine.

Reply:
please restore the changes here if possible. And override the style at the place where you need to use this component